### PR TITLE
Add remaining chapters

### LINKED ISSUE #5: Add 'etude de l'existant' and 'etude fonctionnel' in readme file
demo is in classroom.google.com for reference

### PR DESCRIPTION
**Pull request by Gri-ffin**
- Added étude de l'éexistant.
- Added Etude Fonctionnelle.

### CODE REVIEW

**Comment by Gri-ffin** [comment]
closes #6

File: README.md [hunk 1]
```diff
@@ -37,3 +37,76 @@
...
+            <tr>
+                <td>RG 05</td>
+                <td>Le client peut ajouter qu'un seul like et le retirer</td>
+            </tr>
+            <tr>
+                <td>RG 06</td>
+                <td>L'utilisateur peut liker plusieurs images</td>
+            </tr>
+        </tbody>
+    </table>
```
Comment (AymanLyesri) [inline README.md]: Why did u use **table** tag ??
U need to use the standard table syntax for markdown files *.md in github.
See the [documentation](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)

**Comment by AymanLyesri** [comment]
lemme test it

**Review by AymanLyesri — APPROVED** [review]
nicely done

**Pull request merged**